api/indexer/tx_indexer.go

```diff
@@ -122,36 +120,54 @@ func (*txDBIndexer) storeTransaction(
 	success bool,
 	units fees.Dimensions,
 	fee uint64,
+	outputs [][]byte,
 ) error {
-	v := make([]byte, consts.Uint64Len+1+fees.DimensionsLen+consts.Uint64Len)
-	binary.BigEndian.PutUint64(v, uint64(timestamp))
-	if success {
-		v[consts.Uint64Len] = successByte
-	} else {
-		v[consts.Uint64Len] = failureByte
+	outputLength := 1 // Single byte containing number of outputs
```

Comment:
Note: `GetMaxActionsPerTx()` returns a `uint8`, which enforces the maximum possible number of actions and corresponding results included in a transaction.